MAINTAINERS.md

```diff
@@ -0,0 +1,66 @@
+# SPIRE Helm Charts Maintainership Guidelines and Processes
+
+This document captures the values, guidelines, and processes that the SPIRE Helm Charts project and its maintainers adhere to. All maintainers agree to uphold and abide by the text contained herein.
+
+For a list of active SPIRE Helm Charts maintainers, please see the [CODEOWNERS](CODEOWNERS) file.
+
+## General governance
+
+The SPIRE Helm CHarts project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the [SPIFFE Steering Committee][3] (SSC) the same way that the SPIFFE project and associated maintainers do.
+
+SSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement.
+
+## Maintainer responsibility
+
+SPIRE maintainers adhere to the [requirements and responsibilities][2] set forth in the SPIFFE governance document. They further pledge the following:
+
+* To act in the best interest of the project at all times.
+* To ensure that project development and direction is a function of community needs.
+* To never take any action while hesitant that it is the right action to take.
+* To fulfill the responsibilities outlined in this document and its dependents.
+
+## Guiding principles
+
+The goal of this project is to deliver Helm Charts to the user that are stable, secure, and simple.
```

Comment:
This is really key indeed.

Reply:
The code dump has been through similar review in the 2 years it evolved like this. Also when the dump was done it was reviewed.

Reply:
Please don't think I'm discounting all the hard work that was put into the charts before they were donated. I'm appreciative, I really am.

But, its not the same as review by this community. 

Often there are issues with code drops where one party (usually the contributor) thinks everything is great, and other community members disagree. Its common enough various opensource projects wont accept code drops at all to prevent the kind of conflicts that can result from it.

If I had a chance to review the code as it was getting merged, I would have made many comments on it addressing lots of the issues, before merging. We've now spent lots of time trying to fix all of that in the last number of weeks and there is still more yet to do. A lot o the work has been shifted from the initial contributor to the community now because of this. I'm ok working on fixing it, but need the opportunity to do so.